swan-lake/integration-tools/health-tool.md

```diff
@@ -16,75 +16,128 @@ They are primarily focused on the [Fast Healthcare Interoperability Resources (F
 
 Download a directory containing all FHIR specification definition files. You can download a preferred standard implementation guide from the [FHIR Implementation Guide registry](https://fhir.org/guides/registry/).
 
->**Note:** It is recommended to use a Standard for Trial Use (STU) or higher release level of the implementation guides. Make sure that the downloaded specification archive has the StructureDefinition, ValueSet, and CodeSystem files for the Implementation Guide (IG) resources when extracted.
+>**Note:** It is recommended to use a Standard for Trial Use (STU) or higher release level of the implementation guides. Make sure that the downloaded specification archive has the `StructureDefinition`, `ValueSet`, and `CodeSystem` files for the Implementation Guide (IG) resources when extracted.
+
+## Install the tool
+
+Execute the command below to pull the Health tool from [Ballerina Central](https://central.ballerina.io/search?q=ballerinax%2Fhealth).
+
+```
+$ bal tool pull health
+health:2.0.0 pulled successfully.
+health:2.0.0 successfully set as the active version.
+```
 
 ## Usage
 
-This tool uses the command below to generate a Ballerina package for a given FHIR implementation guide. 
+The Ballerina Health tool support provides the main capabilities below.
+
+- [**Package generation:**](#package-generation) generate a Ballerina package from a given FHIR implementation guide.
+- [**Template generation:**](#template-generation) generate a Ballerina API template from a given FHIR implementation guide.
+
+The general usage of the tool is as follows.
 
 ```
-$ bal health fhir -m <Mode: package> [OPTIONS] <fhir-specification-directory-path>
+$ bal health fhir -m <Mode: package|template> [OPTIONS] <fhir-specification-directory-path>
 ```
 
 >**Note:** Make sure to give the directory path of the downloaded FHIR definitions as the last argument. 
 
-The FHIR resources in the implementation guide will be represented as Ballerina records including the correct cardinality constraints and metadata. FHIR integration developers can leverage the generated package when transforming custom health data into FHIR format without referring to the specification.
-
 ## Command options
```

Comment:
Why having this table while having two separate tables for each category as well?

Reply:
The first table is to give an idea of the general usage of the command with all possible command options and the secondary tables demonstrate the commands that are applicable to the respective use case only.

This is the same format that is used in https://ballerina.io/learn/graphql-tool/.